paper/src/main/java/com/denizenscript/denizen/paper/events/PlayerCompletesAdvancementScriptEventPaperImpl.java

```diff
@@ -0,0 +1,87 @@
+package com.denizenscript.denizen.paper.events;
+
+import com.denizenscript.denizen.utilities.implementation.BukkitScriptEntryData;
+import com.denizenscript.denizen.events.BukkitScriptEvent;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import com.denizenscript.denizencore.objects.core.ListTag;
+import com.denizenscript.denizencore.objects.ObjectTag;
+import com.denizenscript.denizencore.scripts.ScriptEntryData;
+import com.denizenscript.denizencore.utilities.CoreUtilities;
+import net.kyori.adventure.text.Component;
+import net.md_5.bungee.api.ChatColor;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+import org.bukkit.event.player.PlayerAdvancementDoneEvent;
+
+import static com.denizenscript.denizen.paper.PaperModule.stringifyComponent;
+
+public class PlayerCompletesAdvancementScriptEventPaperImpl extends BukkitScriptEvent implements Listener {
+
+    public PlayerCompletesAdvancementScriptEventPaperImpl() {
+        instance = this;
+    }
+    public static PlayerCompletesAdvancementScriptEventPaperImpl instance;
+    public PlayerAdvancementDoneEvent event;
+    @Override

```

Comment:
newlines between things, reference existing events to see where they go, there's a very consistent format of when to or not to use newlines in Denizen source

Reply:
done